e2etests/cypress.config.js

```diff
@@ -26,7 +26,10 @@ module.exports = defineConfig({
 
   e2e: {
 
-    baseUrl: 'https://google.co.in/',
+    baseUrl: 'https://piro.matr.io/',
+    username: '',
+    password: '',
```

Comment:
It cannot be empty and must refer to the secrets key
Also username and password does not appear to be substituted anywhere else in the code.

Reply:
The github secrets will be consumed by setting up github actions. I will be creating a story for github action workflow implementation for piro - all tests